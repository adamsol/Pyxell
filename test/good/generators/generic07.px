
func f(n) def
    for i in 1..n do
        print i
        yield

for _ in f(0) do
    print false

for _ in f(3) do
    print true
