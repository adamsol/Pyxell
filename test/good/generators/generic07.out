1
true
2
true
3
true
